### ISSUE TITLE
Installation-script fails on Raspbian (Pi 3)

### ISSUE THREAD

**Issue by blondbong** [headmelted/codebuilds#14]
First of all thank you for the script :)

I've found out that the installation fails because user root doesn't exist.
Maybe you'll update the requirements on raspbian.

Thank you for your patience and kind regards!

**Comment by headmelted** [comment]
Hi!

Can you attach the output of the installer script to this ticket (or paste it into the textbox?)

This will help me resolve the issue.  Thanks in advance!

**Comment by headmelted** [comment]
Can I ask you to try a fresh install from the script one last time?

I've made changes to the toolchain to use Debian Jessie with emdebian, rather than Ubuntu Ports, which should take the base package versions required back to a release that will hopefully work for everyone (Jessie / Trusty).

**Comment by CameronLamont** [comment]
Thanks for setting this up, much appreciated.

I attempted the script on my RPI3 and it appeared to fail without sudo - there was an error message to that affect but the terminal closed itself before I could get a good look. (I don't even understand how you redirect something into a   .    )

I downloaded the script first and then ran it using sudo and it appears to work.  Not sure if this is the same issue as above.
